src/app/pages/RoflAppDetailsPage/TeeType.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { RoflAppPolicy } from '../../../oasis-nexus/api'
+
+type TeeTypeProps = {
+  policy: RoflAppPolicy
+}
+
+export const TeeType: FC<TeeTypeProps> = ({ policy }) => {
+  const { t } = useTranslation()
+  const usesTdx = policy.quotes?.pcs?.tdx
+  const usesSgx = policy.quotes?.pcs?.sgx
```

Comment:
Can it potentially use both TDX and SGX?

Reply:
not sure, I will take this into account just in case